packages/angular-teleport/.eslintrc.json

```diff
@@ -0,0 +1,40 @@
+{
+  "extends": ["../../.eslintrc.js"],
+  "ignorePatterns": ["!**/*"],
+  "overrides": [
+    {
+      "files": ["*.ts"],
+      "extends": [
+        "plugin:@nrwl/nx/angular",
+        "plugin:@angular-eslint/template/process-inline-templates",
+        "plugin:@cloudflight/angular/recommended"
```

Comment:
The "normal" `recommended` rules should be used in the root of this file not only for `ts` files.

If you really only want the `ts` configuration, then please extend the correct config.   
https://github.com/cloudflightio/cloudflight-eslint-plugin/blob/main/packages/eslint-plugin-angular/README.md

Reply:
Fixed.